certbot/certbot/tests/util.py

```diff
@@ -41,6 +45,41 @@
     from unittest import mock  # type: ignore
 
 
+class DummyInstaller(common.Installer):
```

Comment:
Can we just use the existing `certbot.certbot._internal.plugins.null` class here instead?

Reply:
In fact the purpose of `DummyInstaller`, as a subclass of `certbot.plugins.common.Installer`, is to be used in tests from `certbot.tests.plugins.common_test` whose purpose is to unit test the `certbot.plugins.common` module.

The `Null` plugin just declares the interface `certbot.interface.Installer` and do not implement the features provided by `certbot.common.Installer`. Out-of-the-box this plugin is not working for the unit tests in `certbot.plugins.common`.

Instead of adapting the `Null` plugin with additions that are useless for the other context where it is invoked, I prefered to define a class specifically designed for the unit tests in `certbot.tests.plugins.common_test`.

Reply:
Ah I see. Thanks for the explanation. In that case, this is fine with me.